envs/tracer-service.env

```diff
@@ -0,0 +1,6 @@
+email.service.url=http://email-service:8000/email/send
+spring.datasource.url=jdbc:mysql://docker-mysql-db:3306/tracer_service
+spring.datasource.username=test
+spring.datasource.password=pass
+server.port=10222
+WAIT_HOSTS=mysql:3306
```

Comment:
Je tohle správně? Nemá tam být spíše jméno service v docker compose než plain mysql? Případně používá se to?